scripts/destroy_cluster.sh

```diff
@@ -22,14 +22,25 @@ ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
 
 # shellcheck source=scripts/common.sh
 source "$ROOT/scripts/common.sh"
-
+source "$ROOT/scripts/set-env.sh"
 # Tear down Terraform-managed resources and remove generated tfvars
-cd "$ROOT/terraform/cluster_build" || exit;
+cd "$ROOT/terraform/cluster_build" 
 
 # Perform the destroy
-terraform state rm 'module.kms'
-terraform destroy -input=false -auto-approve
 
-# Remove the tfvars file generated during "make create"
-rm -f "$ROOT/terraform/cluster_build/terraform.tfvars"
-rm -f "$ROOT/terraform/cluster_build/terraform.tfstate"
+if [ "$STATE" == gcs ]; then 
+   cd "${ROOT}/terraform/cluster_build" 
+   sed -i "s/local/gcs/g" backend.tf
+   (cd "${ROOT}/terraform/cluster_build"; terraform init -input=false -backend-config="bucket=$BUCKET")
+   terraform state rm "module.kms" 
+   (cd "${ROOT}/terraform/cluster_build"; terraform destroy -input=false -auto-approve) 
```

Comment:
I think we still need to at least remove the tfvars file during the destroy, ideally we're going to remove the state too.
Also wondering about what happens to the bucket?

Reply:
Add the following
rm -f "$ROOT/terraform/cluster_build/terraform.tfvars"   
gsutil -m rm -r gs://$BUCKET